src/openforms/formio/components/vanilla.py

```diff
@@ -306,11 +349,49 @@ def localize(self, component: SelectComponent, language_code: str, enabled: bool
             return
         translate_options(options, language_code, enabled)
 
+    def build_serializer_field(
+        self, component: SelectComponent
+    ) -> serializers.MultipleChoiceField:
+        validate = component.get("validate", {})
+        required = validate.get("required", False)
+        choices = [
+            (value["value"], value["label"]) for value in component["data"]["values"]
+        ]
+        return serializers.MultipleChoiceField(
+            choices=choices,
+            required=required,
+            allow_blank=not required,
+            allow_null=not required,
```

Comment:
I think the docs for Multiple choice field recommend to use one of allow_blank/allow_null. https://www.django-rest-framework.org/api-guide/fields/#multiplechoicefield
Since here the choices are text, maybe we should only use allow_blank?